src/components/navigation/document-tab-panel.sass

```diff
@@ -11,26 +11,43 @@
     border-color: $classwork-purple
   &.supports
     border-color: $support-blue
-  .tab-list
+  .tab-header-row
+    display: flex
+    flex-direction: row
+    align-items: center
     background-color: white
+    height: $nav-tab-height
+    padding-top: 3px
+    position: relative
+  .tab-list
     margin: 0
     padding: 0
-    height: $nav-tab-height
     display: flex
-    &.my-work
-      background-color: $workspace-teal-light-7
-    &.class-work
-      background-color: $classwork-purple-light-7
-    &.supports
-      background-color: $support-blue-light-7
+    flex: 1 1 auto
+    &:after
+      content: ""
+      border-bottom: solid 1.5px $charcoal-light-1
```

Comment:
I wish Michael wouldn't use these half pixels.  They tend to render odd and causing spacing/sizing issues.